flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStInnerTable.java

```diff
@@ -73,7 +73,7 @@ public interface ForStInnerTable<K, N, V> {
      * @param stateRequest The given stateRequest.
      * @return The corresponding ForSt GetRequest.
      */
-    ForStDBGetRequest<K, N, V> buildDBGetRequest(StateRequest<?, ?, ?> stateRequest);
+    ForStDBGetRequest buildDBGetRequest(StateRequest<?, ?, ?> stateRequest);
```

Comment:
why remove the type?

Reply:
Actually it should not be removed. I added it back.